ruby/game/game_spec.rb

```diff
@@ -5,26 +5,24 @@
 
   it "resets guess count on initialization, prompts user 1 to enter word for user 2 to guess" do
     new_game.word_to_guess
-    expect(new_game.word_to_guess).to eq @word
+    expect(new_game.word_to_guess).to eq ""
   end
 
   it "takes user input and checks guess to see if correct" do
-    new_game.check_input(guess)
-    expect(new_game.check_input(guess)).to eq @is_over
+    expect(new_game.check_input("guess")).to eq ["guess"]
```

Comment:
check input should be returning `true` or `false`

It can still plat a role in updating the array but the array should not be the return.  You should be checking the array contents elsewhere.